aim/web/api/projects/views.py

```diff
@@ -25,9 +24,8 @@ async def project_api():
     return {
         'name': project.name,
         'path': project.path,
-        'tf_enabled': project.tf_enabled,
         'description': project.description,
-        'branches': project.repo.list_branches(),
+        'branches': factory.experiments(),
```

Comment:
@mihran113 here you should iterate over `factory.experiments()` and get dicts from it. Take a look to `web/api/experiments/views.py:get_experiments_list_api` for reference.

Reply:
done ✔️